.github/workflows/draft-new-release.yml

```diff
@@ -0,0 +1,77 @@
+name: "Draft new release"
+
+on:
+  issues:
+    types: [opened, labeled]
+
+jobs:
+  draft-new-release:
+    name: "Draft a new release"
+    runs-on: ubuntu-latest
+    # Only run for issues with a specific title and label. Not strictly required but makes finding the release issue again later easier.
+    # There is also a whitelist that you may want to use to restrict, who can trigger this workflow.
+    # Unfortunately, we cannot create an array on the fly, so the whitelist is just comma-separated.
+    if: startsWith(github.event.issue.title, 'Release version') && contains(github.event.issue.labels.*.name, 'release') && contains('thomaseizinger,bonomat,D4nte,da-kami,luckysori,tcharding,rishflab', github.event.issue.user.login)
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Extract version from issue title
+        run: |
+          TITLE="${{ github.event.issue.title }}"
+          VERSION=${TITLE#Release version }
+
+          echo "::set-env name=RELEASE_VERSION::$VERSION"
+
+      - name: Create release branch
+        run: git checkout -b release/${{ env.RELEASE_VERSION }}
+
+      - name: Update changelog
+        uses: thomaseizinger/keep-a-changelog-new-release@1.1.0
+        with:
+          version: ${{ env.RELEASE_VERSION }}
+
+      - name: Initialize mandatory git config
+        run: |
+          git config user.name "GitHub actions"
```

Comment:
Do we need to add this user to our CLA? 😅 
Let's see what the PR says.

Reply:
That is a good question, probably? We could also make up our own bot user :)

Reply:
Let's see what our CLA check says about this :)